crates/sui-framework/packages/bridge/tests/message_tests.move

```diff
@@ -612,7 +612,7 @@ module bridge::message_tests {
         let ctx = test_scenario::ctx(&mut scenario);
         let coin = coin::mint_for_testing<USDC>(12345, ctx);
         let mut sender = address::to_bytes(sender_address);
-        // double sender which wil make the payload different the 64 bytes
+        // double sender which will make the payload different the 64 bytes
```

Comment:
```suggestion
        // double sender makes the payload different than 64 bytes
```
Suggested this because of the `...different the 64 bytes` at the end doesn't make sense. The rest of it was only because I was already here. Not sure my suggestion states the right info?